Resources/doc/frontend/components.html

```diff
@@ -0,0 +1,148 @@
+<!-- Most of this stuff is just Bootstrap, so check the documentation for more info -->
+
+<!-- LINKS AND BUTTONS -->
+<!-- Button -->
+<!-- Always add mr-2 to icon classes -->
```

Comment:
Zou je niet beter die mr-2 weglaten en dat toevoegen in de css, de php-developers kennen gaan ze dat vergeten, zeker als ze hun iconen van de site van font awesome gaan halen , ze kopieren de code vandaar en vergeten de margin class